src/QueryBuilder.php

```diff
@@ -526,13 +526,11 @@ public function type($type)
      *  );
      * }}}
      *
+     * @param \Elastica\Query\AbstractQuery ...$queries Queries to compare.
```

Comment:
I think this could be a variable number of AbstractQuery or array types.

Reply:
Nothing seems to use/test passing a raw array in, so I don't know if there's different behavior.